demosplan/DemosPlanCoreBundle/Repository/ProcedureRepository.php

```diff
@@ -319,8 +319,9 @@ public function deleteProcedures($procedureIds)
                 }
 
                 $em->remove($em->getReference(Procedure::class, $procedureId));
-                $em->getRepository(ReportEntry::class)
-                    ->deleteByProcedure($procedureId);
+                /** @var ReportRepository $repo */
```

Comment:
Maybe in all of the following cases it would be better to write methods to get specific Repositories and add a return type to each of these methods to ensure correct types.

Reply:
done in [685364f](https://github.com/demos-europe/demosplan-core/pull/2960/commits/685364f97784d85dd520137c6bcee9f67c93e07c)

Reply:
you missed the the cases in lines 369, 382, 1147.
There are even more in this File, maybe you can do all of these but I think it is okay if you only adjust the cases where you added the type via docblock.

Reply:
done in [cdab49a](https://github.com/demos-europe/demosplan-core/pull/2960/commits/cdab49a766a20195437e703013908f7d7e3ce9a5)